draft-yuan-quic-congestion-data.md

```diff
@@ -204,6 +204,14 @@ statistics can be added by registering a value in the appropriate registry (see
 understand, but MAY decline to return protected statistics to a peer if it
 cannot verify that it is willing to share the contained information.
 
+A receiver may get a mesage with multiple occurences of a particular TLV
+value. If the values are identical, the reciever SHOULD ignore them.
+If they differ, and one of the values is protected by an integrity,
+the receiver MAY treat this as an attack and close the connection.
```

Comment:
```suggestion
the receiver SHOULD treat this as an attack and close the connection.
```
I think this path should be a little stronger, particularly as this behavior is already an option below.